icrestimator.py

```diff
@@ -78,16 +81,20 @@ def estimate_lmda(self, q: np.ndarray):
         chassis frame origin.)
         :return: our estimate of ICR as the array (u, v, w)^T.
         """
+        print(f'Estimate_lmda started q_in {q}')
         starting_points = self.select_starting_points(q)
         found = False
         closest_lmda = None
         closest_dist = None
         for lmda_start in starting_points:
+            print(
+                f"iterate over sp, starting dist {np.linalg.norm(self.flip_wheel(q, self.S(lmda_start)))}"
+            )
```

Comment:
We cannot be printing in loops - this was an issue last year and dramatically slowed the robot code.